### PR TITLE
docs: update example of EXPLAIN ANALYZE

### PR DESCRIPTION
**Pull request by waynexia**
## What's Changed in this PR

Update the doc of `EXPLAIN ANALYZE`

## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.9/en/reference/sql/explain.md [hunk 1]
```diff
@@ -35,21 +35,51 @@ The column `plan_type` indicates whether it's a`logical_plan` or `physical_plan`
...
-TODO: explains the output of `ANALYZE`.
+In above example, we got 4 rows of output. Except the last row gives an overall summary, each remains row represents a staged node of the execution plan. The `stage` column indicates the stage number, and the `node` column indicates the node number. The `plan` column explains the plan and related metrics in detail.
+
+### Stage and node
+
+This example is based on a distributed partition table with two regions. We mark different "stage" in a distributed execution plan with stage number, start from 0. And in each stage, we may have multiple "node" to execute the plan in parallel. The node number also starts from 0 in each stage.
+
+### Metrics
+
+Each plan node has its own metrics after the word `metrics`. The metrics may or may not include `output_rows`, `elapsed_compute`, `repart_time`, `fetch_time`, `send_time`, `mem_used` etc. They can be classified into several categories:
```
Comment (killme2008) [inline docs/v0.9/en/reference/sql/explain.md:78]: could you explain all these fields' meaning?

Comment (waynexia) [reply docs/v0.9/en/reference/sql/explain.md:78]: I don't think this is necessary or viable. There are lots of different metrics and most of them have a not bad name describes what it does

Comment (killme2008) [reply docs/v0.9/en/reference/sql/explain.md:78]: To be honest, I don't know. You don't need to list all the fields, but the primary fields deserve to be explained.

Comment (waynexia) [reply docs/v0.9/en/reference/sql/explain.md:78]: The most important one is `elapsed_compute`. Do you want something like this?
```
elapsed_compute: time elapsed in computing
```

File: docs/v0.9/en/reference/sql/explain.md [hunk 2]
```diff
@@ -35,21 +35,51 @@ The column `plan_type` indicates whether it's a`logical_plan` or `physical_plan`
...
+stage: NULL
+ node: NULL
+ plan: Total rows: 0
+4 rows in set (0.002 sec)                
 ```
 
-TODO: explains the output of `ANALYZE`.
+In above example, we got 4 rows of output. Except the last row gives an overall summary, each remains row represents a staged node of the execution plan. The `stage` column indicates the stage number, and the `node` column indicates the node number. The `plan` column explains the plan and related metrics in detail.
+
+### Stage and node
```
Comment (nicecui) [inline docs/v0.9/en/reference/sql/explain.md:72]: 1. is the `stage` a concept only in GreptimeDB? If it is, we need to explain what is the `stage`.
2. It is also seems that the `node` concept is different from the PG's node. for the following example, each plan in PG has a node type, but the node value in Greptime is 0.
If this is different from other db's node, we also need to explain the is the `node` here.
  ```
  *************************** 2. row ***************************
  stage: 1
   node: 0
   plan:  CoalesceBatchesExec: target_batch_size=8192 metrics=[output_rows: 0, elapsed_compute: 4147, ]
    FilterExec: host@2 = host1 metrics=[output_rows: 0, elapsed_compute: 32, ]
      RepartitionExec: partitioning=RoundRobinBatch(32), input_partitions=8 metrics=[repart_time: 8, fetch_time: 230515, send_time: 256, ]
        SeqScan: region=5471788335105(1274, 1), partition_count=0 (0 memtable ranges, 0 file ranges) metrics=[output_rows: 0, mem_used: 0, ]
  ```

Comment (nicecui) [reply docs/v0.9/en/reference/sql/explain.md:72]: Or we need to explain the relationships between the distributed architecture and stage and node.

Comment (waynexia) [reply docs/v0.9/en/reference/sql/explain.md:72]: Both `stage` and `node` are only in the scope of this `EXPLAIN ANALYZE`. They are just for distinguish sub-plans

File: docs/v0.9/en/reference/sql/explain.md [hunk 3]
```diff
@@ -35,21 +35,51 @@ The column `plan_type` indicates whether it's a`logical_plan` or `physical_plan`
...
+
+This example is based on a distributed partition table with two regions. We mark different "stage" in a distributed execution plan with stage number, start from 0. And in each stage, we may have multiple "node" to execute the plan in parallel. The node number also starts from 0 in each stage.
+
+### Metrics
+
+Each plan node has its own metrics after the word `metrics`. The metrics may or may not include `output_rows`, `elapsed_compute`, `repart_time`, `fetch_time`, `send_time`, `mem_used` etc. They can be classified into several categories:
+- Timer
+  - `elapsed_compute`, `repart_time`, `fetch_time`, `send_time` etc.
+  - Shows the elapsed time in nanosecond. Different timer corresponding different time span.
+  - Notice that for plans that have multiple partitions, the timer is usually summed across all partitions.
```
Comment (nicecui) [inline docs/v0.9/en/reference/sql/explain.md:82]: The explanation is not clear enough and needs to be more explicit.

File: docs/v0.9/en/reference/sql/explain.md (hunk 1 above)
Comment (killme2008) [reply docs/v0.9/en/reference/sql/explain.md:78]: Yes, something like that, including `fetch_time`, `send_time` etc.

File: docs/v0.9/en/reference/sql/explain.md (hunk 3 above)
Comment (waynexia) [reply docs/v0.9/en/reference/sql/explain.md:82]: Could you please figure out which specific place is not clear?

File: docs/v0.9/en/reference/sql/explain.md (hunk 2 above)
Comment (waynexia) [reply docs/v0.9/en/reference/sql/explain.md:72]: >Or we need to explain the relationships between the distributed architecture and stage and node.

Okay

File: docs/v0.9/en/reference/sql/explain.md (hunk 3 above)
Comment (nicecui) [reply docs/v0.9/en/reference/sql/explain.md:82]: I'm not sure, I have two questions when I read this line:
1. what's the detailed summing logic across all partitions?
2. do all the timers use the summing logic? it doesn't seem reasonable for `fetch_time` or `send_time`

Comment (waynexia) [reply docs/v0.9/en/reference/sql/explain.md]: please checkout [7bdfabe](https://github.com/GreptimeTeam/docs/pull/1114/commits/7bdfabe260168ca62094ab1fc12119b3e40ae793)

"summing" is using "add" operation to reduce many numbers into one, is there any ambiguity?

File: docs/v0.9/en/reference/sql/explain.md [hunk 4]
```diff
@@ -35,21 +35,59 @@ The column `plan_type` indicates whether it's a`logical_plan` or `physical_plan`
...
 ```
 
-TODO: explains the output of `ANALYZE`.
+In above example, we got 4 rows of output. Except the last row gives an overall summary, each remains row represents a staged node of the execution plan. The `stage` column indicates the stage number, and the `node` column indicates the node number. The `plan` column explains the plan and related metrics in detail.
+
+### Stage and node
+
+This example is based on a distributed partition table with two regions. We mark different "stage" in a distributed execution plan with stage number, start from 0. And in each stage, we may have multiple "node" to execute the plan in parallel. The node number also starts from 0 in each stage. These two words are only used here currently.
+
+### Metrics
```
Comment (fengjiachun) [inline docs/v0.9/en/reference/sql/explain.md:76]: I don't think this section is necessary, especially the time-related metrics. They do not encompass all the times on the entire execution timeline and belong to different plans individually. Forcing an explanation would only confuse users more. To properly explain what they represent, I believe we should write separate documents for each plan in the future and detail the significance of these metrics.

**Review by fengjiachun — APPROVED** [review]
LGTM

**Review by nicecui — COMMENTED** [review]
please translate it to Chinese and copy to nightly

**Comment by waynexia** [comment]
> please translate it to Chinese and copy to nightly

Please checkout https://github.com/GreptimeTeam/docs/pull/1114/commits/dd5dd54561f1fbc89d40a915bd95eca4499aad63

File: docs/v0.9/zh/reference/sql/explain.md [hunk 5]
```diff
@@ -33,21 +33,42 @@ EXPLAIN SELECT * FROM monitor where host='host1';
...
+ plan: Total rows: 0
+4 rows in set (0.002 sec)       
 ```
 
-<!-- TODO: explains the output of `ANALYZE`. -->
+在上面的例子中，我们得到了 4 行输出。除了最后一行给出了总体摘要，每一行都代表了执行计划的一个阶段节点。`stage` 列表示阶段编号，`node` 列表示节点编号。`plan` 列详细说明了计划和相关指标。
+
+## Stage 和 node
+
+这个例子基于一个有两个区域的分布式分区表。我们使用阶段编号在分布式执行计划中标记不同的“阶段”，从 0 开始。在每个阶段中，我们可能有多个“节点”并行执行计划。节点编号在每个阶段中也从 0 开始。这两个词目前只在这里使用。以下是第二阶段中有 2 个“阶段”和 3 个“节点”的示例：
```
Comment (fengjiachun) [inline docs/v0.9/zh/reference/sql/explain.md:72]:
```suggestion
这个例子基于一个有两个 region 的分布式分区表。我们使用阶段编号在分布式执行计划中标记不同的“阶段”，从 0 开始。在每个阶段中，我们可能有多个“节点”并行执行计划。节点编号在每个阶段中也从 0 开始。这两个词目前只在这里使用。以下是第二阶段中有 2 个“阶段”和 3 个“节点”的示例：
```

File: docs/v0.9/zh/reference/sql/explain.md [hunk 6]
```diff
@@ -33,21 +33,42 @@ EXPLAIN SELECT * FROM monitor where host='host1';
...
+ plan: Total rows: 0
+4 rows in set (0.002 sec)       
 ```
 
-<!-- TODO: explains the output of `ANALYZE`. -->
+在上面的例子中，我们得到了 4 行输出。除了最后一行给出了总体摘要，每一行都代表了执行计划的一个阶段节点。`stage` 列表示阶段编号，`node` 列表示节点编号。`plan` 列详细说明了计划和相关指标。
+
+## Stage 和 node
+
+这个例子基于一个有两个 region 的分布式分区表。我们使用阶段编号在分布式执行计划中标记不同的“阶段”，从 0 开始。在每个阶段中，我们可能有多个“节点”并行执行计划。节点编号在每个阶段中也从 0 开始。这两个词目前只在这里使用。以下是第二阶段中有 2 个“阶段”和 3 个“节点”的示例：
```
Comment (nicecui) [inline docs/v0.9/zh/reference/sql/explain.md:72]:
```suggestion
这个例子基于一个有两个 region 的分布式分区表。我们使用 stage 编号在分布式执行计划中标记不同的 stage，从 0 开始。在每个 stage 中，我们可能有多个 node 并行执行计划。node 编号在每个 stage 中也从 0 开始。以下是有 2 个 stage 和 3 个 node 的示例：
```

File: docs/v0.9/en/reference/sql/explain.md [hunk 7]
```diff
@@ -35,21 +35,42 @@ The column `plan_type` indicates whether it's a`logical_plan` or `physical_plan`
...
+ plan: Total rows: 0
+4 rows in set (0.002 sec)                
 ```
 
-TODO: explains the output of `ANALYZE`.
+In above example, we got 4 rows of output. Except the last row gives an overall summary, each remains row represents a staged node of the execution plan. The `stage` column indicates the stage number, and the `node` column indicates the node number. The `plan` column explains the plan and related metrics in detail.
+
+### Stage and node
+
+This example is based on a distributed partition table with two regions. We mark different "stage" in a distributed execution plan with stage number, start from 0. And in each stage, we may have multiple "node" to execute the plan in parallel. The node number also starts from 0 in each stage. These two words are only used here currently. Here is an with 2 "stage" and 3 "node" in the second stage:
```
Comment (nicecui) [inline docs/v0.9/en/reference/sql/explain.md:74]:
```suggestion
This example is based on a distributed partition table with two regions. We mark different "stage" in a distributed execution plan with stage number, start from 0. And in each stage, we may have multiple "node" to execute the plan in parallel. The node number also starts from 0 in each stage. These two words are only used here currently. Here is an example about 3 "node" with 2 "stage":
```

**Review by nicecui — COMMENTED** [review]
please translate copy to nightly

**Comment by nicecui** [comment]
outdated

**Pull request closed without merging**